packages/docs/docs/launch-configuration.md

```diff
@@ -146,8 +154,12 @@ command used to build the app. Example below:
       "type": "radon-ide",
       "request": "launch",
       "name": "Radon IDE panel",
-      "buildScript": {
-        "android": "npm run build:ftp-fetch-android"
+      "customBuild": {
+        "android": { "buildScript": "npm run build:ftp-fetch-android" }
```

Comment:
This isn't a script, so maybe we should rename to `buildCommand`? On the other hand in `package.json` you also have `scripts` sections where you can put commands and not just scripts.